docs/data-process.rst

```diff
@@ -99,6 +99,107 @@ LIMITATIONS
    recognised will not be processed.
 
 
+Modularisation / Include with Merge
+-----------------------------------
+
+A common use case for include is to merge a list read from an include
+file into the current list, (or similarly to merge a map/object read from an
+include file into the current map/object). We can tell the processor with the
+``MERGE`` keyword.
```

Comment:
Might be worth continuing this with:
We can tell the processor with the ``MERGE`` keyword, which if not specified defaults to false.

Reply:
Done.